packages/storybook-test/src/ExampleBodyText.tsx

```diff
@@ -0,0 +1,6 @@
+import type { PropsWithChildren, ReactNode } from 'react';
+import './ExampleBodyText.css';
+
+export const ExampleBodyText = ({ children }: PropsWithChildren) => <div className="example-body-text">{children}</div>;
```

Comment:
Dit is niet per se nodig, je kunt ook er `globals.storyRootClassname` voor naar `voorbeeld-theme example-body-text` zetten zodat je deze extra `<div>` niet in je HTML voorbeeld ziet. Als je hem juist wel wilt zien dan is dit prima.